macros/Cargo.toml

```diff
@@ -1,17 +1,26 @@
 [package]
-name = "num-macros"
-version = "0.1.33"
 authors = ["The Rust Project Developers"]
-license = "MIT/Apache-2.0"
-homepage = "https://github.com/rust-num/num"
-repository = "https://github.com/rust-num/num"
+description = "Numeric syntax extensions"
 documentation = "http://rust-num.github.io/num"
+homepage = "https://github.com/rust-num/num"
 keywords = ["mathematics", "numerics"]
-description = "Numeric syntax extensions"
+license = "MIT/Apache-2.0"
+name = "num-macros"
+repository = "https://github.com/rust-num/num"
+version = "0.1.33"
 
-[lib]
-name = "num_macros"
-plugin = true
+[dependencies]
+quote = "0.1.3"
+syn = "0.6.0"
 
 [dev-dependencies]
-num = { path = "..", version = "0.1" }
+
+[dev-dependencies.num]
+path = ".."
+version = "0.1"
+
+[lib]
+crate-type = ["rustc-macro"]
+name = "num_macros"
+rustc-macro = true
```

Comment:
None, but it isn't harmful in any way and this helps with visibility (you do not need to check `Cargo.toml` for crate type, it is already there).